etc/ci.sh

```diff
@@ -69,14 +76,17 @@ testmanuals)
         QUIT_GAP(0);
 GAPInput
 
+    TESTMANUALSPASS=yes
     for ch in tst/testmanuals/*.tst
     do
-        bin/gap.sh -q -L testmanuals.wsp --cover $COVDIR/$(basename $ch).coverage <<GAPInput
-        TestManualChapter("$ch");
-        QUIT_GAP(0);
-GAPInput
+        TestManualChapter $ch
```

Comment:
A simpler way to implement this (which possibly avoids the need of creating `TestManualChapter`), using the `||` operator:
```
bin/gap.sh -q -L testmanuals.wsp --cover $COVDIR/$(basename $ch).coverage <<GAPInput || TESTMANUALSPASS=no
...
```
Or of course also
```
echo "TestManualChapter(\"$1\"); QUIT_GAP(0);" | bin/gap.sh ... || TESTMANUALSPASS=no
```